components/scream/scripts/test-all-scream

```diff
@@ -81,6 +82,10 @@ def _main_func(description):
     success = tas.test_all_scream()
 
     print("OVERALL STATUS: {}".format("PASS" if success else "FAIL"))
+    if not is_repo_clean():
```

Comment:
You can remove this if you make the change above.

Reply:
Uhm, I think the warning is still meaningful, no? Even if the user explicitly passed '--keep-tree', we want to remind him/her that the PASS/FAIL depends on the uncommitted changes.